bin/citgm-all.js

```diff
@@ -17,6 +20,16 @@ yargs = commonArgs(yargs)
     alias: 'f',
     type: 'boolean',
     description: 'Ignore flaky flags. Don\'t ignore any failures.'
+  })
+  .option('parallel', {
+    alias: 'j',
+    type: 'number',
+    description: 'Number of tests to run in parallel'
+  })
+  .option('autoParallel', {
```

Comment:
I would personally rather have a separate parameter instead of having to type `-j=auto`. using `-J` to auto calculate makes more sense to me so I would argue against this